cmd/azmount/fuse.go

```diff
@@ -113,3 +135,38 @@ func (f File) Read(ctx context.Context, req *fuse.ReadRequest, resp *fuse.ReadRe
 	resp.Data = data
 	return err
 }
+
+func (f *File) ReadAll(ctx context.Context) ([]byte, error) {
```

Comment:
What is this function for?

Reply:
Kapil Vaswani actually wrote this part (I did the testing/doc updates), but I believe that has to do with how to implement the go FUSE interface following this example: https://github.com/bazil/fuse/blob/master/examples/hellofs/hello.go

Reply:
It's required to comply with the interface but isn't used internally.